masterdata/returnRequest/schema.json

```diff
@@ -0,0 +1,211 @@
+          "required": true
+        },
+        "city": {
+          "type": "string",
+          "required": true
+        },
+        "state": {
+          "type": "string",
+          "required": true
+        },
+        "country": {
+          "type": "string",
+          "required": true
+        },
+        "zipCode": {
+          "type": "string",
+          "required": true
+        },
+        "addressType": {
+          "type": "string",
+          "required": true,
+          "enum": ["pickup-point", "customer-address"]
+        },
+        "returnLabel": {
+          "type": "string"
+        }
+      }
+    },
+    "refundPaymentData": {
+      "type": "object",
+      "required": true,
+      "properties": {
+        "refundPaymentMethod": {
+          "type": "string",
+          "required": true,
+          "enum": ["bank", "card", "giftCard"]
+        },
+        "iban": {
+          "type": "string"
+        },
+        "accountHolderName": {
+          "type": "string"
+        },
+        "paidWithCc": {
+          "type": "boolean"
+        }
+      }
+    },
+    "items": {
+      "type": "array",
+      "required": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "orderItemIndex": { "type": "integer", "required": true },
+          "quantity": { "type": "integer", "required": true },
+          "verifiedItems": { "type": "integer" },
+          "reasonCode": { "type": "string", "required": true },
+          "reason": { "type": "string", "required": true },
+          "condition": {
+            "type": "string",
+            "required": true,
+            "enum": [
+              "newWithBox",
+              "newWithoutBox",
+              "usedWithBox",
+              "usedWithoutBox"
+            ]
+          }
+        }
+      }
+    },
+    "dateSubmitted": { "type": "string", "format": "date-time" },
+    "userComment": { "type": "string", "maxLength": 300 },
+    "refundData": {
+      "type": ["object", "null"],
+      "required": true,
+      "properties": {
+        "invoiceNumber": { "type": "string", "required": true },
+        "invoiceValue": { "type": "integer", "required": true },
+        "refundedItemsValue": { "type": "integer", "required": true },
+        "refundedShippingValue": { "type": "integer" },
+        "giftCard": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string"
+            },
+            "code": {
+              "type": "string"
+            }
+          }
+        },
+        "items": {
+          "type": "array",
+          "required": true,
+          "items": {
+            "type": "object",
+            "properties": {
+              "orderItemIndex": { "type": "integer", "required": true },
+              "quantity": { "type": "integer", "required": true },
+              "restockFee": { "type": "integer" }
+            }
+          }
+        }
+      }
+    },
```

Comment:
`refundData` will be populate when the process is done - invoice to return items is completed.